src/script/components/MessagesList/Message/MemberMessage.tsx

```diff
@@ -176,6 +176,16 @@ const MemberMessage: React.FC<MemberMessageProps> = ({
           )}
         </>
       )}
+      <div className="message-header">
+        <div className="message-header-icon" />
+        <p className="message-header-label">{t('conversationNewConversation')}</p>
+      </div>
+      <div className="message-header">
+        <div className="message-header-icon message-header-icon--svg text-foreground">
+          <Icon.Info />
+        </div>
+        <p className="message-header-label">{t('conversationUnverifiedUserWarning')}</p>
+      </div>
```

Comment:
refactor of this 
![image](https://user-images.githubusercontent.com/78490891/220103855-61d6dd23-7d7b-4c0d-9dff-b8dd8bee06ad.png)